descriptors/ModuleDescriptor-template.json

```diff
@@ -5,7 +5,7 @@
   "requires": [
     {
       "id": "oai-pmh",
-      "version": "1.1"
+      "version": "1.1 2.0"
```

Comment:
I understand we want to support `1.1 `and `2.0`. But if we only have `2.0`, it is still backward compatible with earlier versions? If so, why not only have `2.0`?

Reply:
There's no need to force the dependency on 2.0...  If someone wants to run this with 1.1 they should be able to.  If we only put 2.0 here that won't be allowed.